elide-core/src/main/java/com/yahoo/elide/Elide.java

```diff
@@ -380,6 +405,7 @@ protected ElideResponse handleRequest(boolean isReadOnly, User user,
                 requestScope.getPermissionExecutor().printCheckStats();
             }
 
+            elideCompleteQuery(requestScope, response);
```

Comment:
This needs to happen in the finally block.  It might make sense for accept query to actually return a closeable of some kind so that we can accept the query inside the try statement itself (in the same way we create the transaction).

We can then call process and complete on the closeable returned from accept.